view/stache/intermediate_and_imports.js

```diff
@@ -26,7 +26,7 @@ steal("can/view/stache/mustache_core.js", "can/view/parser",
 			attrStart: function( attrName ){
 				if(attrName === "from") {
 					inFrom = true;
-				} else if(attrName === "as") {
+				} else if(attrName === "as" || attrName === "export-as") {
```

Comment:
This one: https://github.com/bitovi/canjs/blob/rename-as-1966/view/stache/intermediate_and_imports.js#L36